pkg/madmin/info-commands.go

```diff
@@ -198,6 +198,44 @@ func (adm *AdminClient) DataUsageInfo() (DataUsageInfo, error) {
 	return dataUsageInfo, nil
 }
 
+// AccountAccess contains information about
+type AccountAccess struct {
+	AccountName string `json:"accountName"`
+	Read        bool   `json:"read"`
+	Write       bool   `json:"write"`
+	Other       bool   `json:"other"`
```

Comment:
Rather than other as should say Custom I think

Reply:
Not sure where is the usage here @vadmeste ?

Reply:
> Not sure where is the usage here @vadmeste ?

Basically this will gather all permissions that are no read or write. I think it is helpful so the admin can still quickly notice that one user has a permission (setting bucket notification, etc..)